src/GraphQL/Introspection/__EnumValue.cs

```diff
@@ -12,11 +12,11 @@ public __EnumValue()
                 "a placeholder for a string or numeric value. However an Enum value is " +
                 "returned in a JSON response as a string.";
 
-            Field(f => f.Name);
-            Field(f => f.Description, nullable: true);
+            Field(f => f.Name).Description(null);
+            Field(f => f.Description, nullable: true).Description(null);
 
             Field<NonNullGraphType<BooleanGraphType>>("isDeprecated", resolve: context => (!string.IsNullOrWhiteSpace(context.Source?.DeprecationReason)).Boxed());
-            Field(f => f.DeprecationReason, nullable: true);
+            Field(f => f.DeprecationReason, nullable: true).Description(null);
```

Comment:
Without `.Description(null)`, the schema pulls the description from the xml comments that were added.

Reply:
Funny that we have to hide the description. It is right?

Reply:
There was no description in the schema before I added the xml comment.  It's default behavior for a `Field` method call with an expression to pull the description from the xml comment, I guess.  So to override this default, I set it to null, so that the schema retains the prior description (none).  I'm open to suggestions, but I had not intended to add a description to the schema in this PR, but only to add xml comments to the code.

Reply:
The spec says obvious descriptions can be omitted. We _can_ add these descriptions if we _want_.

Reply:
Regardless, I would not auto-import schema descriptions from the xml code comments.

Reply:
Let's leave them `null`.